packages/atomic/custom-elements.json

```diff
@@ -0,0 +1,8811 @@
+{
```

Comment:
? what's this file

Reply:
I have no clue, seems like it was automatically generated when I added .nx to gitignore:

eda6232

I'll delete it.